program.py

```diff
@@ -61,29 +61,29 @@ def get_token(endpoint):
 
     # we can't short circuit it, so we must go retreive it.
 
-    discoveryUrl = requests.get(
+    discovery_URL = requests.get(
```

Comment:
is capitalizing 'URL' proper snake_case?

Reply:
I changed 'URL' to lower case